tests/cleo_tests/compile.bat

```diff
@@ -0,0 +1,4 @@
+@REM Compile all .txt files in the current directory to .s files using Sanny Builder 4
+@REM Usage: set SANNY="path\to\sanny.exe" && compile.bat
+
+for /f "delims=" %%i in ('dir /b /s *.txt') do %SANNY% --compile "%%i" "%%~pi%%~ni.s" --no-splash --mode sa_sbl
```

Comment:
It would be good to delete all compiled files first. So in case of compilation fail there will be no leftovers masking it.

Reply:
Also use echo to print current file into console, so some progress is visible
adding 
'echo Done.` and `pause' gives chance to verify results of the process

Reply:
done